rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -191,6 +193,57 @@ rclcpp::init(int argc, char * argv[])
 #endif
 }
 
+std::vector<std::string>
+rclcpp::init_and_remove_ros_arguments(int argc, char const * const argv[])
+{
+  rclcpp::init(argc, argv);
+  return rclcpp::remove_ros_arguments(argc, argv);
+}
+
+std::vector<std::string>
+rclcpp::remove_ros_arguments(int argc, char const * const argv[])
+{
+  rcl_allocator_t alloc = rcl_get_default_allocator();
+  rcl_arguments_t parsed_args;
```

Comment:
Need to zero initialize this before calling `rcl_get_zero_initialized_arguments()`. This could be the source of the random segfault.

Reply:
Possible cause of #458